shared/languages/pythonPyodide.ts

```diff
@@ -109,9 +109,9 @@ function exceptionRewriter(msg: string, _: string) {
   });
 }
 
-export const PYTHON = {
-  name: 'Python',
-  defaultPath: 'python3',
+export const PYTHON_PYODIDE = {
+  name: 'Python (pyodide)',
+  defaultPath: 'python3-pyodide',
```

Comment:
This default path should not change, this is about where actual-python3 gets called when running in desktop/server mode.